src/qminer/qminer_core.h

```diff
@@ -3272,6 +3291,9 @@ class TIndex {
     /// Index RecId using given keys and words. Words are extracted by tokenizing the given string.
     void DeleteTextPos(const int& KeyId, const TUInt64V& WordIdV, const uint64& RecId);
 
+    /// method responsible for indexing or deleting words from index
```

Comment:
Is the method pointer really so complicated? I would see creating pairs of new objects, and putting them to a vector pretty unnecessary in this case.